naga/src/front/glsl/parser_tests.rs

```diff
@@ -263,7 +263,7 @@ fn declarations() {
             &Options::from(ShaderStage::Vertex),
             r#"
         #version 450
-        layout(push_constant)
+        layout(immediate_data)
```

Comment:
```suggestion
        layout(push_constant)
```

This needs to stay the same as it's coming from GLSL.